internal/vendor/wiz.go

```diff
@@ -326,6 +355,7 @@ type Project struct {
 	Entrypoints            []*ProjectEntrypoint            `json:"entrypoints"`
 	ID                     string                          `json:"id"`
 	Identifiers            []string                        `json:"identifiers"`
+	KubernetesClusterLinks []*ProjectKubernetesClusterLink `json:"kubernetesClustersLinks"`
```

Comment:
Rename from KubernetesClusterLinks to KubernetesCluster**s**Links (clusters is plural).